### ISSUE TITLE
Custom CSS

### ISSUE THREAD

**Issue by lmr2391** [sernst/cauldron#20]
A great feature would be the option to include own css-files to provide some custom styling. Is that something, that's on your roadmap?

**Comment by sernst** [comment]
Yes, more flexible custom CSS is definitely on the roadmap. You can hack with scoped style tags when you use `cd.display.html()`, but more general inclusion planned as well.
